### PR TITLE
data-ga tagging updates

### PR DESCRIPTION
**Pull request by jacekkolasa**

### CODE REVIEW

File: src/components/tableV2/features/globalControls.js [hunk 1]
```diff
@@ -11,13 +11,14 @@ const GlobalControls = ({
...
 }) => {
+  const wordsCount = searchValue?.split(" ").filter(x => !!x).length
   return (
     <Flex width="100%" zIndex={10} background="mainBackground" padding={[0, 0, 4]}>
       {handleSearch && (
         <Flex width={{ max: 100, base: "40%" }}>
           <SearchInput
             data-testid="table-global-search-filter"
-            data-ga={`${dataGa}::search::table-filter`}
+            data-ga={`${dataGa}::search-${wordsCount}-words::table-filter`}
```
Comment (burbuli8ra) [inline src/components/tableV2/features/globalControls.js:21]: Why is this needed here?

Comment (jacekkolasa) [reply src/components/tableV2/features/globalControls.js:21]: @burbuli8ra it's addressing [this requirement](https://github.com/netdata/product/issues/3064#user-content-functions-low-prio)
```
 Search bar interactions - how many users use this search bar? ideally capturing the words used
```

Comment (burbuli8ra) [reply src/components/tableV2/features/globalControls.js:21]: I mean this will constantly be updated while user types, so I cannot really understand how it is helpful for analytics

Comment (jacekkolasa) [reply src/components/tableV2/features/globalControls.js:21]: @burbuli8ra i'm not sure which tools we do have at our disposal in PostHog or GA, i thought that when it will be deployed, we can experiment and modify it if it won't be useful.

**Review by burbuli8ra — APPROVED** [review]
